tools/deps/wingetopt/src/getopt.h

```diff
@@ -0,0 +1,95 @@
+#ifndef __GETOPT_H__
```

Comment:
Since this file and the getopt.c are taken from another repository and are left untouched (they might get updated/fixed/improved) they will have their own code style which it is very likely to be different than the one set for this project. The CI will complain about this (build will fail) so these files should be written on the formatting ignore list:
https://github.com/analogdevicesinc/aditof_sdk/blob/master/.clangformatignore

Reply:
fixed